yarn.lock

```diff
@@ -16947,7 +16947,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"typescript@npm:5.1.5":
+"typescript@npm:5.1.5, typescript@npm:^5.1.3":
```

Comment:
hmm is this a bug? why adding 5.1.3 reference back while removing 5.1.3 declarations...